.changeset/dull-masks-occur.md

```diff
@@ -0,0 +1,19 @@
+---
+'@backstage/plugin-catalog-react': minor
+---
+
+`entityRouteParams` and `useEntityRoute` both extract entity route parameters for use with an entity route reference.
```

Comment:
Changesets are for end users to understand the "what" of the changes made, not so much the "why" or targeting developers of the code itself.

This changeset could be condensed down to something along the lines of

```
`entityRouteParams` now also accepts entity refs, and can help with encoding the resulting parameters.
```

Reply:
Updated to suggestion.